### PR TITLE
refactor(CircleCi): change JDK to version 11 in CircleCi job

### PR DESCRIPTION
**Pull request by bekobarna**

### CODE REVIEW

**Comment by markkovari** [comment]
Hallod @csadesz 2 napja all a PR ;) ne stoppold

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,20 +15,23 @@ jobs:
       - checkout
 
       - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "build.gradle" }}
-          - v1-dependencies-
+          key: v1-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - restore_cache:
+          key: v1-gradle-cache-{{ checksum "build.gradle" }}
```
Comment (csadesz) [inline .circleci/config.yml]: Could be under one restore_cache as 'keys'

File: .circleci/config.yml [hunk 2]
```diff
@@ -15,20 +15,23 @@ jobs:
...
       - save_cache:
           paths:
-            - ~/.gradle
-          key: v1-dependencies-{{ checksum "build.gradle" }}
+            - ~/.gradle/wrapper
+          key: v1-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - save_cache:
+          paths:
+            - ~/.gradle/caches
+          key: v1-gradle-cache-{{ checksum "build.gradle" }}
```
Comment (csadesz) [inline .circleci/config.yml]: Referring to the CricleCI documentation the order should be like 

save_cache:
   key:
   paths:

**Comment by bekobarna** [comment]
fixed based on request

**Pull request merged**